x-pack/plugins/lens/public/xy_visualization/axis_settings_popover.tsx

```diff
@@ -236,6 +286,178 @@ export const AxisSettingsPopover: React.FunctionComponent<AxisSettingsPopoverPro
           />
         </>
       )}
+      {localExtent && setExtent && (
+        <>
+          <EuiSpacer size="s" />
+          <EuiFormRow
+            display="rowCompressed"
+            fullWidth
+            label={i18n.translate('xpack.lens.xyChart.axisExtent.label', {
+              defaultMessage: 'Bounds',
+            })}
+            helpText={
+              hasBarOrAreaOnAxis && localExtent.mode !== 'custom'
+                ? i18n.translate('xpack.lens.xyChart.axisExtent.disabledDataBoundsMessage', {
+                    defaultMessage:
+                      "Bounds of bar and area series always have to include zero and can't be fit to the data",
```

Comment:
Since we already notify users about the zero inclusion in the custom inputs, I think we can shorten this text by removing that redundant text. Perhaps something like:

```suggestion
                      "The selected chart cannot be fit to the data bounds.",
```

Reply:
I think we should tell the user which charts _can_ be fit to data bounds, otherwise they just a have a problem, but no obvious solution. I renamed to "Only line charts can be fit to the data bounds"